.travis.yml

```diff
@@ -1,8 +1,6 @@
 dist: trusty
 
 language: java
-node_js:
-  - 10
```

Comment:
why have you deleted this config? it's essential for successful build because travis is using 8 version of node by default, which may produce errors